frontend/src/pages/start/Start.tsx

```diff
@@ -1,14 +1,29 @@
 import React, { FunctionComponent } from "react";
-import { Redirect, Route, Switch } from "react-router-dom";
+import { Redirect, Route, RouteProps, Switch } from "react-router-dom";
 import Main from "./main/Main";
 import Configuration from "./configuration/Configuration";
+import Layout from "components/layout/Layout";
+
+const RouteWithLayout = (props: RouteProps) => (
+  <Layout>
+    <Route {...props} />
+  </Layout>
+);
```

Comment:
Nit: Could we reuse the `RouteWithLayout` component that's already defined in `App.tsx` (i think)?

Reply:
I did not want to use it, because this would result in a circular import. Will make it non redundant.